packages/trpc/server/routers/viewer/organizations/create.handler.ts

```diff
@@ -148,9 +148,9 @@ export const createHandler = async ({ input, ctx }: CreateOptions) => {
         organization: {
           create: {
             name,
-            ...(!IS_TEAM_BILLING_ENABLED && { slug }),
+            ...(IS_TEAM_BILLING_ENABLED ? { slug } : {}),
```

Comment:
@zomars @alannnc note that this unintentional change on the condition ("!" removed) created some unexpected behavior when creating orgs locally as there was no slug registered for new orgs, adding it back in an upcoming PR